contents/blog/customer-churn-analysis-guide.md

```diff
@@ -0,0 +1,296 @@
+---
+date: 2023-02-07
+title: "Retention vs Churn Rate: A complete guide to churn analysis"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - andy-vandervell
+featuredImage: ../images/blog/posthog-marketing/marketing-hog.jpg
+featuredImageType: full
+category: Product growth
+tags:
+  - Product analytics
+  - Product metrics
+  - Guides
+  - Explainers
+---
+
+You know what's cooler than gaining new customers? Retaining them.
+
+In this guide to customer churn analysis, I'm exploring churn rate and retention rate:
+- How are they different?
+- How do you measure them? 
+- Which one would win a fight?<sup>*</sup>
+
+These and other useful questions (like [how to reduce churn](#how-to-improve-retention-and-reduce-churn) will be answered. Let's get to it.
+
+<sup>(*strictly speaking, they'd cancel each other out... read the next section to understand why)</sup>
+
+## What's the difference between retention rate and churn rate?
+
+Not much, really: **churn rate** is the percentage of customers who **stop using** your product during a given period of time; **retention rate** is the percentage of **existing customers** who **continue using** your product. 
+
+In other words, if you have 1,000 customers and 100 of them cancel their subscription, your monthly churn rate would be 10% and your retention rate would be 90%. That's it.
+
+Simple, right? Yes, but these definitions ignore three important points:
+
+1. There are numerous ways to measure and analyze them
+1. Just knowing what they are won't help you improve them
+1. While both are important, in my view **retention rate** is more useful (most of the time)
+
+I'll explore these points further as we go. See [Takeaways on churn analysis](#takeaways-on-churn-analysis) for a TL;DR.
+
+## Understanding churn rate
+
+Here's the simplest formula for churn rate:
+
+> **Customer churn rate** = (Churned customers during a specific period / Total customers at the start of that period) x 100
+
+Where churn customers equal:
+
+> **Churned customers** = (Customers at the start of the period - Customers at the end of the period) + New customers acquired in that period
+
+The majority of companies should track churn monthly and/or annually. Those interested in tracking churn daily or weekly are better off using a [cohort retention table](#how-to-read-and-use-a-cohort-retention-table). More on those later.
+
+> 🚨 **Important:** Do not confuse monthly and annual churn rates. Monthly churn compounds into annual churn – i.e. a monthly churn rate of 5% equals a massive 46% annual churn rate. You can't just multiply your monthly churn rate by 12 to get your annual number, either. See [how to convert monthly churn to annual](#how-to-convert-monthly-churn-into-annual-rate) in the appendix.
+
+### What is a good churn rate?
+
+Benchmarking churn rate is tricky because:
```

Comment:
I hate it when I read an article and the answer to a question I have is 'it's complicated' followed by a bunch of caveats. I think this section would be stronger if it went something like: 

It depends:
- For consumer apps, e.g. social media, it's X
- For B2B software with small contract sizes (e.g. $10-100/month) it's Y
- etc.

It sounds like you have the data, so why not surface a bit more of it, vs. having people run off to another article

Reply:
e.g. this is way better https://www.lennysnewsletter.com/p/what-is-good-retention-issue-29